modules/tinymce/src/plugins/lists/main/ts/actions/ToggleList.ts

```diff
@@ -347,6 +365,13 @@ const toggleSingleList = (editor: Editor, parentList: HTMLElement | null, listNa
       flattenListSelection(editor);
     } else {
       const bookmark = Bookmark.createBookmark(editor.selection.getRng());
+      if (isCustomList(parentList)) {
+        parentList.classList.forEach((cls, _, classList) => {
+          if (!/\btox\-/.test(cls)) {
```

Comment:
I assume this is some rewrite from being a className match on `\b` since `\b` is a word boundary a single class name should never have it. So this would likely be better written as `cls.startsWith('tox-')`. Since we want to remove all classes matching `tox-<something>` right.

Reply:
Done!